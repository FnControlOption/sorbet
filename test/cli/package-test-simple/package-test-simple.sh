cd test/cli/package-test-simple || exit 1

../../../main/sorbet --silence-dev-message --stripe-packages --stripe-mode . 2>&1

echo
echo "TODO(ngroman) Move this error enforcement into Sorbet"
