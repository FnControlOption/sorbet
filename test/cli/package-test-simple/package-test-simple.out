No errors! Great job.

TODO(ngroman) Move this error enforcement into Sorbet
